### PR TITLE
Merge Dev into Main

### PR DESCRIPTION
**Pull request by weissercn**

### CODE REVIEW

**Comment by weissercn** [comment]
Done
- simplified environments and vectorized environments by removing duplicate code
- added KTO experiments
- updated yamls with human_openness_to_influence and realism score

File: influence_benchmark/experiments/hyperparameter_sweep.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def train_loop(config=None):
 
         total_time = end_time - start_time
         print("final_preference", final_preference)
-        score = final_preference[1][-1]
+        score = final_preference[1][-1]  # TODO check
```
Comment (micahcarroll) [inline influence_benchmark/experiments/hyperparameter_sweep.py:69]: TODO check

File: influence_benchmark/stats/preferences_per_iteration.py [hunk 2]
```diff
@@ -16,17 +16,30 @@ def load_trajectories(trajectory_path: Path) -> pd.DataFrame:
 
 
 def get_top_n_trajectories(trajectory_path: Path, num_chosen_trajectories: int) -> List[Dict]:
+    return get_func_n_trajectories(trajectory_path, num_chosen_trajectories, pd.DataFrame.nlargest)
```
Comment (micahcarroll) [inline influence_benchmark/stats/preferences_per_iteration.py:19]: We can probably remove this and always use the best_worst method. Speed gains are probably miniscule

**Review by micahcarroll — APPROVED** [review]
LGTM broadly, left some comments which i think should probably be addressed before merging

File: influence_benchmark/RL/SFT.py [hunk 3]
```diff
@@ -38,8 +52,10 @@ def train_sft():
     sft_config.gradient_checkpointing_kwargs = args.g_c_kwargs
     sft_config.dataset_text_field = "text"
 
+    print(args.lora_path)
```
[the remark this answers is not in the document]
Comment (carolius) [reply influence_benchmark/RL/SFT.py:55]: I think we do want to, but I'll add a "Lora path: ",

File: influence_benchmark/RL/KTO.py [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+from influence_benchmark.agent.agent import Agent
+from influence_benchmark.root import PROJECT_DATA, PROJECT_ROOT
+from influence_benchmark.stats.preferences_per_iteration import analyze_run, get_best_worst_n_trajectories
+from influence_benchmark.utils.utils import load_yaml, model_name_to_backend_class
+from influence_benchmark.vectorized_environment.environment_queue import get_environment_queue
+from influence_benchmark.vectorized_environment.vectorized_environment import VectorizedEnvironment
+
+
+class KTO:
```
[the remark this answers is not in the document]
Comment (carolius) [reply influence_benchmark/RL/KTO.py:19]: Yeah i think we could make both inherit from a more general class

File: influence_benchmark/stats/preferences_per_iteration.py (hunk 2 above)
Comment (carolius) [reply influence_benchmark/stats/preferences_per_iteration.py:19]: I think removing it makes some other code less nice to read?

**Pull request merged**